<html><h1>Validation FAILED.</h1><p>Build Time: 21 minutes 52 seconds 888 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/7aa4bfd3c78f654417186fb3cf88f2b0f347f6e9">7aa4bfd3c78f654417186fb3cf88f2b0f347f6e9</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :x:<pre><code>     [echo] java.lang.Exception: Found 1 formatting issues:
     [echo] 1: ./commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CommerceOrderGenerator.java expected:&lt;...up.getCompanyId());
     [echo] []
     [echo] 		searchContext.set...&gt; but was:&lt;...up.getCompanyId());
     [echo] [		]
     [echo] 		searchContext.set...&gt;
     [echo] 
     [echo] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:438)
     [echo] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:266)
     [echo] 
     [echo] FAILURE: Build failed with an exception.
     [echo] 
     [echo] * What went wrong:
     [echo] Execution failed for task ':checkSourceFormatting'.
     [echo] &gt; Process 'command '/opt/java/jdk1.8.0_121-x64/bin/java'' finished with non-zero exit value 1
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkSourceFormatting'.
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [echo] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [echo] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [echo] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [echo] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)</code></pre></li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-16.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/81//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-16.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/81//testReport">here</a>.</h5></div></html>

--- a/commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CommerceOrderGenerator.java
+++ b/commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CommerceOrderGenerator.java
@@ -165,7 +165,7 @@ public class CommerceOrderGenerator {
 
 		List<CommerceAddress> commerceAddresses =
 			_commerceAddressLocalService.getCommerceAddresses(
-				groupId, CommerceAccount.class.getName(),
+				CommerceAccount.class.getName(),
 				commerceAccount.getCommerceAccountId(), 0, 1, null);
 
 		if (commerceAddresses.isEmpty()) {
@@ -177,6 +177,9 @@ public class CommerceOrderGenerator {
 		}
 
 		CommerceAddress commerceAddress = commerceAddresses.get(0);
+
+		commerceOrder.setShippingAddressId(
+			commerceAddress.getCommerceAddressId());
 
 		// Commerce shipping options
 
@@ -419,7 +422,8 @@ public class CommerceOrderGenerator {
 		Group group = _groupLocalService.getGroup(groupId);
 
 		searchContext.setCompanyId(group.getCompanyId());
-		searchContext.setGroupIds(new long[] {group.getGroupId()});
+
+		searchContext.setAttribute("commerceChannelGroupId", groupId);
 
 		return searchContext;
 	}

--- a/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/util/CPDefinitionHelperImpl.java
+++ b/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/util/CPDefinitionHelperImpl.java
@@ -132,7 +132,7 @@ public class CPDefinitionHelperImpl implements CPDefinitionHelper {
 			CPDefinitionIndexer.FIELD_PUBLISHED, Boolean.TRUE);
 
 		searchContext.setEnd(end);
-		searchContext.setGroupIds(new long[] {groupId});
+		searchContext.setAttribute("commerceChannelGroupId", groupId);
 
 		QueryConfig queryConfig = searchContext.getQueryConfig();
 

